_includes/footer.html

```diff
@@ -0,0 +1,39 @@
+                        <p><a href="https://github.com/vacp2p" target="_blank">Check out the code</a></p>
+                        <p>It all lives in Github. Here, test it out, XYZ</p>
+                    </li>
+                    <li>
+                        <a href="https://forum.vac.dev/" target="_blank"><svg width="60" height="59" viewBox="0 0 60 59" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M31.0942 58.2734C23.9642 58.2734 17.4449 55.6679 12.4045 51.3578L0 53.0525L4.79223 41.2247C3.07059 37.4276 2.18189 33.3059 2.1859 29.1367C2.1859 13.0459 15.1286 0 31.0942 0C47.0586 0 60 13.0459 60 29.1367C60 45.2305 47.059 58.2734 31.0942 58.2734ZM46.8786 29.0534V28.9722C46.8786 20.5645 40.9494 14.5692 30.726 14.5692H19.6843V43.7059H30.5624C40.8652 43.7063 46.8786 37.4607 46.8786 29.0534ZM30.8478 36.5473H27.6181V21.7299H30.8478C35.5912 21.7299 38.7397 24.4342 38.7397 29.0983V29.1794C38.7397 33.8827 35.5912 36.5473 30.8478 36.5473Z" fill="#121212"/></svg></a>
+                        <p><a href="https://forum.vac.dev/" target="_blank">Join the research forum</a></p>
+                        <p>Find answers, pose questions, and join the open forum on Disqus</p>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank"><svg width="60" height="69" viewBox="0 0 60 69" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M39.8057 32.5714C39.8057 34.6629 38.2629 36.3771 36.3086 36.3771C34.3886 36.3771 32.8114 34.6629 32.8114 32.5714C32.8114 30.48 34.3543 28.7657 36.3086 28.7657C38.2629 28.7657 39.8057 30.48 39.8057 32.5714ZM23.7943 28.7657C21.84 28.7657 20.2971 30.48 20.2971 32.5714C20.2971 34.6629 21.8743 36.3771 23.7943 36.3771C25.7486 36.3771 27.2914 34.6629 27.2914 32.5714C27.3257 30.48 25.7486 28.7657 23.7943 28.7657ZM60 7.06286V68.5714C51.3624 60.9383 54.1248 63.465 44.0914 54.1371L45.9086 60.48H7.02857C3.15429 60.48 0 57.3257 0 53.4171V7.06286C0 3.15429 3.15429 0 7.02857 0H52.9714C56.8457 0 60 3.15429 60 7.06286ZM50.2286 39.5657C50.2286 28.5257 45.2914 19.5771 45.2914 19.5771C40.3543 15.8743 35.6571 15.9771 35.6571 15.9771L35.1771 16.5257C41.0057 18.3086 43.7143 20.88 43.7143 20.88C35.57 16.4163 26.003 16.4155 18.1029 19.8857C16.8343 20.4686 16.08 20.88 16.08 20.88C16.08 20.88 18.9257 18.1714 25.0971 16.3886L24.7543 15.9771C24.7543 15.9771 20.0571 15.8743 15.12 19.5771C15.12 19.5771 10.1829 28.5257 10.1829 39.5657C10.1829 39.5657 13.0629 44.5371 20.64 44.7771C20.64 44.7771 21.9086 43.2343 22.9371 41.9314C18.5829 40.6286 16.9371 37.8857 16.9371 37.8857C17.4415 38.2388 18.2732 38.6964 18.3429 38.7429C24.1299 41.9837 32.3502 43.0454 39.7371 39.9429C40.9371 39.4971 42.2743 38.8457 43.68 37.92C43.68 37.92 41.9657 40.7314 37.4743 42C38.5029 43.3029 39.7371 44.7771 39.7371 44.7771C47.3143 44.5371 50.2286 39.5657 50.2286 39.5657Z" fill="#121212"/></svg></a>
+                        <p><a href="#" target="_blank">Chat with us</a></p>
```

Comment:
Can we replace this call to action with Telegram and https://t.me/vacp2p?

Reply:
Done